src/includes/class-wordlift.php

```diff
@@ -1074,8 +1074,6 @@ private function load_dependencies() {
 			function () use ( &$that, $schemaorg_property_service ) {
 
 				/** Services. */
-				// Create the configuration service.
-				$api_service = new Wordlift_Api_Service();
```

Comment:
I took a peek at the api_service
[16:18](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663679924698319)
you removed the whole new … which makes sense
[16:18](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663679933375879)
but it looks that the service expects to be instantiaed
[16:18](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663679938227869)
(the code is badly written)
[16:19](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663679941152069)
image.png
 
image.png


[16:19](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663679991022449)
the get_instance method should actually check whether the static $instance var has been initialized or not
[16:20](https://wordlift-io.slack.com/archives/D043L7K0LM6/p1663680006539349)
something like
		if ( self::$instance === null ) {
			self::$instance = new self();
		}

		return self::$instance;